app/styles/components/CardList.scss

```diff
@@ -45,7 +45,6 @@
       width: 100%;
       height: 219px;
       position: relative;
-      background-color: #D8D8D8;
```

Comment:
The initial reason why this was added was to show a poster as a placeholder for images. In other words, in the case of slow poster image loading, the background color would temporarily serve a placeholder. Maybe moving it to a darker shade would be a solution?